lib/ansible/modules/user.py

```diff
@@ -1375,7 +1375,9 @@ def chown_homedir(self, uid, gid, path):
                 for d in dirs:
                     os.chown(os.path.join(root, d), uid, gid)
                 for f in files:
-                    os.chown(os.path.join(root, f), uid, gid)
+                    full_path = os.path.join(root, f)
+                    if not os.path.islink(full_path):
+                        os.chown(os.path.join(root, f), uid, gid)
```

Comment:
did you want to reuse that var?
```suggestion
                        os.chown(full_path, uid, gid)
```

Reply:
no, just forgot to restore after a revert